insights/client/client.py

```diff
@@ -52,14 +53,15 @@ def get_console_handler():
     else:
         target_level = logging.INFO
 
-    stdout_handler = logging.StreamHandler(sys.stderr)
+    stdout_handler = logging.StreamHandler(sys.stdout)
```

Comment:
Why not put all logging to stderr and only stderr?  This actually breaks `--to-stdout` since stdout will have log messages mixed in.  Also, under normal operation, error messages get printed to the console twice.

Reply:
Thanks for the changes here.  Shouldn't logging go to stderr, though?